SayTheirNames/Source/Service/Networking/BaseNetworkUrl.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Say Their Names
+//
+//  Created by evilpenguin.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+
+class BaseNetworkUrl {
+    typealias UrlString = String
+    
+    class var base: UrlString {
+        #if DEV
+            return "https://saytheirnames.dev"
+        #elseif QA
+            return "https://saytheirnames.qa"
+        #elseif LOCAL
+            return "https://localhost"
+        #else
+            return "https://saytheirnames.dev" // switch to PROD
+        #endif
+    }
+}
```

Comment:
This could be simplified to smth like:
```
enum Environment {
    static let serverURLString = {
        #if ... 
    }()
}
```
and used like `Environment.serverURLString`

Reply:
Fixed.